setup.py

```diff
@@ -437,8 +437,29 @@ def find_tbb(tbb_root=None, tbb_include=None, tbb_link=None):
             long_description=readme,
             long_description_content_type='text/x-rst',
             url='https://github.com/glotzerlab/freud',
+            download_url='https://pypi.org/project/freud-analysis/',
```

Comment:
Is there a way to mention conda-forge here as well? Would probably be good to have both if possible.

Reply:
@tommy-waltmann I don't know of any packages that advertise their conda-forge packages in `setup.py` -- I usually reference NumPy, Pandas, scikit-learn, and similar tools to know what is worth mentioning (or not). However, this reminded me to add `project_urls` as in this PR to signac: https://github.com/glotzerlab/signac/pull/334/files